src/Lib/LanguagesLib.php

```diff
@@ -612,6 +612,7 @@ public static function languagesInTatoeba()
                 'mni' => __d('languages', 'Meitei'),
                 'ayl' => __d('languages', 'Libyan Arabic'),
                 'ood' => __d('languages', "O'odham"),
+                'ckb' => __d('languages', 'Central Kurdish (Soranî)',
```

Comment:
The closing parenthesis is still missing on this line (before the last comma).

Reply:
Thank you!